ckanext/switzerland/logic.py

```diff
@@ -293,14 +293,23 @@ def ogdch_showcase_search(context, data_dict):
     the number of datasets in each showcase in the output.
     '''
     user = tk.get_action('get_site_user')({'ignore_auth': True}, {})
-    context.update({'user': user['name'], 'for_view': True})
```

Comment:
If we need to remove this key from the context, we also need to update the docstring, which says:
```
    with 'for_view'=True
    so that the ckanext-showcase before_view method is called. This includes
    the number of datasets in each showcase in the output.
```